spec/features/item_data_page_spec.rb

```diff
@@ -1,8 +1,19 @@
 require 'spec_helper'
 
 describe 'item', type: :feature, js: true, vcr: true do
-  # it '/works?query=climate' do
-  #   visit '/works?query=climate'
-  #   expect(page).to have_css 'h4.results'
-  # end
+  describe 'metrics tests' do
+
+    it 'when metrics should exist has metrics' do
+      visit '/works/10.7272/q6g15xs4'
+      expect(page).to have_css 'span.metrics-views'
+      expect(page).to have_css 'span.metrics-downloads'
+    end
+
+    it 'when metrics should not exist doesnt have has metrics' do
```

Comment:
The test for the chart should be very similar. I'm putting the charts within a divs with 

-           .row#downloads-chart
-           .row#view-chart